src/biogeochem/CNPhenologyMod.F90

```diff
@@ -954,9 +971,14 @@ subroutine CNSeasonDecidPhenology (num_soilp, filter_soilp       , &
                   days_active(p) = days_active(p) + fracday
                   if (days_active(p) > 355._r8) pftmayexist(p) = .false.
                end if
-
+               ! use 15 hr max to ~11hours in temperate regions
                ! only begin to test for offset daylength once past the summer sol
-               if (ws_flag == 0._r8 .and. dayl(g) < crit_dayl) then
+               crit_daylat=54000-360*(65-grc%latdeg(g))
+               if (crit_daylat < crit_dayl) then
+                  crit_daylat = crit_dayl
+               end if
```

Comment:
AD spinup has a function that has the latitude dependence get_spinup_latitude_term in SoilBiogeochemStateType.F90. It is smoothly varying as opposed to here where you have different behavior at the 45 degree mark. Numerics are better with smoothly varying things rather than a big change at a given threshold.

Reply:
54000, 360, and 65 should become parameters on the paramsfile. Do any of these relate to other parameters? And what are some suggestions on names?

Reply:
hi, just a quick comment on crit_dayl. It was the previous ~11 hour cut off that was based on a study using NDVI over the United States. I think maintaining that value for temperate latitudes is probably important. I don't have observational evidence that the critical day length should continue to decrease past 11 hours in temperate latitudes.

Reply:
comments in the code about what each value / parameter means would be helpful.  E.g. 54000 = minutes of daylight (15hrs) of daylength @ XX degrees north.   

If these are parameters we can maintain backwards capability.